### PR TITLE
Enhance workflow

### PR DESCRIPTION
**Pull request by ekohl**
This uses the helper scripts in {deb,rpm}/develop to gather all packages that need to be bumped. This is then inserted as a matrix of packages to bump.

The bumping jobs are now reduced to setup (install dependencies + git checkout) and calling a bumping helper with the name and version.

It also fixes the rpmdev-packager to be a minimal script in itself. An alias can't be used inside another script.

It also renames the workflow to bump_packages.yml since it now has no notion of gems. It just understands packages and versions. All specifics are inside the {deb,rpm}/develop branches inside helpers. This opens the path to bumping other things, such as Python packages or Ansible collections.

The only downside that remains is that every run generates a new commit which is then force pushed. This causes CI to run for every branch which can become quite expensive if it runs daily.

### CODE REVIEW

**Comment by ekohl** [comment]
Example output: https://github.com/ekohl/foreman-packaging/actions/runs/1564370425

File: .github/workflows/bump_packages.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      matrix:
+        include: ${{ fromJson(needs.rpm_list.outputs.matrix) }}
+
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y rpm git-annex python3-semver
+        sudo gem install gem2rpm --no-document
+        sudo curl --create-dirs -o /usr/local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
+        echo 'echo "$GIT_AUTHOR_NAME <$GIT_AUTHOR_EMAIL>"' | sudo tee /usr/local/bin/rpmdev-packager
```
Comment (ekohl) [inline .github/workflows/bump_packages.yml:38]: Technically we can get `rpmdev-packager` from pagure in the same way as above, but that doesn't respect `GIT_AUTHOR_*` env variables.

**Comment by ekohl** [comment]
Updated. I'd say this now works mostly. I also wonder if I should just squash the history and submit it as my own PR now.

File: .github/workflows/bump_packages.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
+name: Update packages
+on:
+  workflow_dispatch:
+env:
+  GIT_AUTHOR_NAME: ${{ github.actor }}
+  GIT_AUTHOR_EMAIL: ${{ github.actor }}@users.noreply.github.com
```
Comment (ekohl) [inline .github/workflows/bump_packages.yml:6]: We can argue whether we should set the author to the one who triggered or just set a bot name. I'm leaning to the latter.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @ekohl ! :+1:

**Pull request merged**